.github/ISSUE_TEMPLATE/-issue-number---problem-description-.md

```diff
@@ -0,0 +1,19 @@
+---
+name: "[Issue number - problem description]"
+---
+
+## Extension
+- [ ] Server
+- [ ] Client
+
+## Java Version
+Please, provide the output of `java --version`.
+
+## Quarkus Version
+Please, specify the version of Quarkus used.
+
+## Description
+Please, provide a detailed description of the issue or the feature request.
```

Comment:
```suggestion
<!-- Please, provide a detailed description of the issue or the feature request. -->
```

Reply:
@gmalheiro it doesn't seem like you accepted this one.

Reply:
@gmalheiro can you please comment this line before we get it merged?

Reply:
@hbelmiro  Sure!